source/features/profile-gists-link.tsx

```diff
@@ -42,6 +42,12 @@ async function init(): Promise<void> {
 	);
 
 	const navigationBar = (await elementReady('.UnderlineNav-body'))!;
+
+	// If the button already exists, skip adding it
+	const isGistButtonAlreadyAdded = !!select(".UnderlineNav-body [data-tab-item='rgh-gists-item']");
+	if(isGistButtonAlreadyAdded) return;
```

Comment:
Both these fixes get the job done but they're just band-aid early returns right now - I have a feeling the real issue is that the event handlers that these init functions are adding aren't being torn down when navigating away causing them to build up over time